### DISCUSSION TITLE
Add own App

### DISCUSSION THREAD

**Discussion by ghost** [leokhoa/laragon#616] [Laragon's Articles & How-tos]
I want to add my own apps to quick Add and that works really good with composer like

#WinterCMS 
WinterCMS=composer create-project wintercms/winter

My problem is, that the installation should be done everytime in a special folder like winter so that the local structure is  website.dev/winter

How can i do that?

**Comment by Pen-y-Fan** [comment]
Try copying the Laravel line, and modifying **larave/laravel** to **wintercms/winter**. 

It looks like %s is the placeholder to create the project in the folder you are prompted for:

```ini
# Laravel

Laravel=composer create-project laravel/laravel %s --prefer-dist

#WinterCMS
WinterCMS=composer create-project wintercms/winter %s --prefer-dist
```

**Reply by ghost** [reply]
Thanks but this i've tested before i write the post because it doesn't do what i want :-)

When i machte this and create a new  website like test  with this it is doesn't create a folder named winter in the domainfolder. I try to create a new website and in this websitefolder it should be create the wintercms with composer so that the structure looks like

website (url is than website.dev)
 |- winter
      |- (all installed/created folders and files from the composer
      
  The way I've done it manually so far is that I create an empty website and then enter composer create-project wintercms/winter /winter via the console. Then everything is installed in the website/winter. I would like to simplify this :-)
  
  I don't know if this is even possible in Laragon and the local Windows environment

**Reply by kstingel** [reply]
did you try modifying `usr/tpl/VirtualHost.tpl` and creating the structure you want there?
